ocfweb/docs/docs/staff/staffvm/ocfweb.md

```diff
@@ -0,0 +1,26 @@
+[[!meta title="Running ocfweb on your Staff VM"]]
+
+If you want to work on [`ocfweb`][0] on your Staff VM, it isn't as
+simple as cloning the repo and running `make dev`, unfortunately.
+In order to do so, you will need to add the `ocfweb` dev config to
+your Staff VM's Puppet configuration, so Puppet can install
+`ocfweb`'s dependencies. To do this, clone the [puppet repo][1] or
```

Comment:
yeah redis and celery don't have system dependencies. this is strictly to provide the dev config.

there *are* some system dependencies (libcrack2-dev or whatever) but the dev config doesn't attempt to provide all of these (and probably shouldn't, since the list is hard to maintain)